osu.Game/Skinning/Editor/SkinSelectionHandler.cs

```diff
@@ -333,22 +255,75 @@ private void applyOrigin(int hash)
         private Quad getSelectionQuad() =>
             GetSurroundingQuad(SelectedBlueprints.SelectMany(b => b.Item.ScreenSpaceDrawQuad.GetVertices().ToArray()));
 
-        private void applyAnchor(int hash)
+        private void applyAnchor(Anchor anchor)
```

Comment:
I will suggest renaming this to `applyCustomAnchor` or something, given that this turns on `OverridesClosestAnchor`.